apps/web/lib/slots.ts

```diff
@@ -15,26 +15,29 @@ export type GetSlots = {
   frequency: number;
   workingHours: WorkingHours[];
   minimumBookingNotice: number;
+  eventLength: number;
 };
 export type WorkingHoursTimeFrame = { startTime: number; endTime: number };
 
 const splitAvailableTime = (
   startTimeMinutes: number,
   endTimeMinutes: number,
-  frequency: number
+  frequency: number,
+  eventLength: number
 ): Array<WorkingHoursTimeFrame> => {
   let initialTime = startTimeMinutes;
   const finalizationTime = endTimeMinutes;
   const result = [] as Array<WorkingHoursTimeFrame>;
   while (initialTime < finalizationTime) {
     const periodTime = initialTime + frequency;
-    result.push({ startTime: initialTime, endTime: periodTime });
+    if (initialTime + eventLength <= finalizationTime)
```

Comment:
Here the same thing as above, explicit if conditionals
`if (currentTimePlusEventLength <= finalizationTime)`
Probably with a better name constant than the one I mentioned